packages/components/src/SegmentedControl/SegmentedControl.module.css

```
@@ -3,6 +3,8 @@
   grid-template-columns: repeat(var(--segmentedControl--option-count), 1fr);
   align-items: center;
   position: relative;
+  height: var(--space-larger);
```

```suggestion
  min-height: 32px;
```

We (semi-)recently aligned on [moving away from using spacing tokens for sizing](https://atlantis.getjobber.com/?path=/docs/guides-frontend-styleguide--docs#size-width-and-height) - you can reference Button as an example of what that looks like in practice:

https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Button/Button.module.css#L11

https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Button/Button.module.css#L184

---

Shoulda remembered this! Done here fdc3bd61a2c24f8cb3185930cedcd94fade6fd6e